app/controllers/api/v1/api_controller.rb

```diff
@@ -1,5 +1,5 @@
 module Api::V1
-  class ApiController < ApplicationController
+  class ApiController <  Api::V1::UtilityController
```

Comment:
Rather than inherit from `UtilityController` you can put the functions directly into `ApiController` and delete UtilityController. The functions are specific to the API so that's ok.